docs/reference/data-streams/lifecycle/tutorial-manage-new-data-stream.asciidoc

```diff
@@ -0,0 +1,148 @@
+[role="xpack"]
+[[tutorial-manage-new-data-stream]]
+=== Tutorial: Create a data stream with a lifecycle
+
+preview::[]
+
+To create a data stream with a built-in lifecycle, follow these steps:
+
+. <<create-index-template-with-lifecycle>>
+. <<create-data-stream-with-lifecycle>>
+. <<retrieve-lifecycle-information>>
+
+[discrete]
+[[create-index-template-with-lifecycle]]
+==== Create an index template
+
+A data stream requires a matching <<index-templates,index template>>. The same way you use index
+templates for mappings and index settings you can use them to configure the data stream lifecycle, the respective
+template is called `lifecycle`. You can define an index template that sets a lifecycle as follows:
```

Comment:
The part after the comma ("the respective template...") surprised me. Should hat be a separate sentence?

Reply:
I will rephrase it, to see if it gets better. What about:

> A data stream requires a matching <<index-templates,index template>>. The same way you use index
> templates for mappings and index settings you can use them to configure the data stream lifecycle by setting the field called `lifecycle`.